example/test_sqlflow/DNN_model/tutorial_dnn_Titanic.ipynb

```diff
@@ -0,0 +1,297 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Classify Titanic Dataset Using DNNClassifer\n",
+    "\n",
+    "This tutorial demonstrates how to\n",
+    "1. train a DNNClassifer on Titanic dataset.\n",
+    "1. use trained DNNClassifer to predict the class of the passenger's survival status.\n",
+    "\n",
+    "## The Dataset\n",
+    "\n",
+    "The Titanic dataset comes from Kaggle(https://www.kaggle.com/c/titanic). The task is to predict which passengers survived the tragedy.\n",
+    "\n",
+    "We have used feature engineering to preprocess the raw data and create new features. Finally, we got the `train.csv` and `test.csv` files. The feature engineering file is `titanic_preprocessing.py` in the current directory.\n",
```

Comment:
> Finally, we got the `train.csv` and `test.csv` files. The feature engineering file is `titanic_preprocessing.py` in the current directory.\n",


You can execute the Python script `[titanic_preprocessing.py](./titanic_preprocessing)`  to preprocess the raw data, and got `train.csv` and `test.csv` finally.